pyfixest/estimation/feols_.py

```diff
@@ -1436,6 +1436,7 @@ def predict(
         atol: float = 1e-6,
         btol: float = 1e-6,
         type: str = "link",
+        compute_stdp: bool = False
     ) -> np.ndarray:
```

Comment:
Btw, I think that a few unit tests fail because the code assumes that `predict` returns a `np.ndarray` - if we change to a `pd.DataFrame` (which I think makes most sense), we need to change these. 

Alternatively, we could change to `-> Union[np.ndarray, pd.DataFrame]` and simply return the numpy array if `compute_stdp = False`.